Gemfile

```diff
@@ -69,6 +65,12 @@ else
   gem 'govuk_frontend_toolkit', '3.1.0'
 end
 
+if ENV['SLIMMER_DEV']
+  gem 'slimmer', path: '../slimmer'
+else
+  gem 'slimmer', '8.1.0', github: "alphagov/slimmer", branch: "obey-strict-mode"
```

Comment:
A new version of slimmer will need to be released with the change we want to be used here before this PR can be merged.